### PR TITLE
7.0.0 alpha1, alpha2 and beta1 release notes

### PR DESCRIPTION
**Pull request by colinsurprenant**
I did not include notes for changes that were also include in v6.

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -30,21 +30,22 @@ This section summarizes the changes in the following releases:
...
+* Cleanup Ruby gems dependencies https://github.com/elastic/logstash/pull/10171[#10171]
+* Ensure compatibility of module data with ES and Kibana 7.0 https://github.com/elastic/logstash/pull/10356[#10356]
+* Rename x-pack monitoring and management config option .url and .ca to .hosts and .certificate_authority https://github.com/elastic/logstash/pull/10380[#10380]
+* BUGFIX: building of deb and rpm artifacts https://github.com/elastic/logstash/pull/10396[#10396]
 
 [[logstash-7-0-0-alpha2]]
 === Logstash 7.0.0-alpha2 Release Notes
 
-Placeholder for release notes
+* No user-facing changes
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc]: I think we can add here at least:

```
* d5b5f9a4f - Add Zing JVM to the list of supported garbage collectors. (4 months ago)
* e4159d0fb - BUGFIX: setting a list config to empty array should not result in nil (4 months ago)
```

Comment (colinsurprenant) [reply docs/static/releasenotes.asciidoc]: Are you sure? https://github.com/elastic/logstash/pull/10051 was also merged into 6.x and https://github.com/elastic/logstash/pull/10179 in 6.5 and 6.x
I am a bit confused about what we should add in these release notes; I omitted all that were already previously merged into a v6 branch ...

Comment (jsvd) [reply docs/static/releasenotes.asciidoc]: Ah, I see, what you say makes sense. Nevermind then

**Comment by colinsurprenant** [comment]
added plugin changes for alpha1, alpha2, beta1. will do the rest later or tomorrow.

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -3,12 +3,22 @@
...
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-0-0]]
+=== Logstash 7.0.0 Release Notes
+
+coming[7.0.0]
+
+The list combines release notes from the 7.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: I added a placeholder and "coming soon" label for 7.0.0.  I left the 7.0.0 section empty for now. No point copying all of the other content over and then having to make any corrections in two places.

**Comment by colinsurprenant** [comment]
I added rc1 and rc2!

**Pull request merged**